tests/test-rest-users-controller.php

```diff
@@ -1079,8 +1096,16 @@ protected function check_user_data( $user, $data, $context, $links ) {
 			$this->assertEquals( $user->last_name, $data['last_name'] );
 			$this->assertEquals( $user->nickname, $data['nickname'] );
 			$this->assertEquals( $user->user_email, $data['email'] );
-			$this->assertEquals( $user->allcaps, $data['capabilities'] );
-			$this->assertEquals( $user->caps, $data['extra_capabilities'] );
+			if ( 'object' === gettype( $data['capabilities'] ) ) {
+				$this->assertEquals( new stdClass(), $data['capabilities'] );
+			} else {
+				$this->assertEquals( $user->allcaps, $data['capabilities'] );
+			}
```

Comment:
It's because associative arrays equal objects in json.  That is why there is the difference.  The weird part is that this test only flagged for failing on running multisite.xml.